src/Symfony/Component/Security/Core/Authorization/Voter/Decorator/Weight.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Security\Core\Authorization\Voter\Decorator;
+
+use InvalidArgumentException;
```

Comment:
@dosten This change came from the fabbot CS checks, so I guess that's the right way.